src/components/NotificationDetail.tsx

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+
+type NotificationProps = {
+  key: number;
+  read: boolean;
```

Comment:
read って動詞っぽくて関数と勘違いしそうなので、 isRead とかにするのはいかがでしょうか？

```suggestion
  isRead: boolean;
```

Reply:
https://github.com/tatata05/next-app/pull/10/commits/e187f8805578eeca47d202dfda36142641465660
修正しました！